unison-cli/src/Unison/Codebase/Editor/UriParser.hs

```diff
@@ -297,7 +350,7 @@ nameSegment =
 gitTreeishSuffix :: P Text
 gitTreeishSuffix = P.label "git treeish" . P.try $ do
   void $ C.char ':'
-  P.takeWhile1P (Just "not close paren") (/= ')')
+  P.takeWhile1P (Just "not close paren") (\c -> c /= ')' && not (isSpace c))
```

Comment:
Was this to support whitespaces inside the parens in the config file, or for something else?